### ISSUE TITLE
get accuracy error

### ISSUE THREAD

**Issue by Pharaun85** [KevinMusgrave/pytorch-metric-learning#269]
Hello!
I don't know if it's a misconception and I'm not using it right but the AccuracyCalculator is giving me an error. 
I am trying to calculate the accuracy in both training and validation, for which I do the following: 

```
acc_metric = AccuracyCalculator(include=('mean_average_precision_at_r',))

data = sample['data']  # RGB images
        label = sample['label']
        if torch.cuda.is_available() and args.use_gpu:
            data = data.cuda()
            label = label.cuda()

        embeddings = model(data)
        if miner is not None:
            hard_pairs = miner(embeddings.squeeze(), label)  # El miner no funciona y no da ningun error
            loss = criterion(embeddings.squeeze(), label, hard_pairs)
        else:
            loss = criterion(embeddings.squeeze(), label)


        acc_dict = acc_metric.get_accuracy(embeddings.squeeze().cpu(), embeddings.squeeze().cpu(), label.cpu(),
                                           label.cpu(),
                                           embeddings_come_from_same_source=True)


        acc = acc_dict['mean_average_precision_at_r']
```

the error that appears is that bool object does not have 'ndim' attribute.

Could it be because I have not been able to install fass-gpu correctly?

Thanks

**Comment by KevinMusgrave** [comment]
What is the shape of ```embeddings``` and ```label```?

**Comment by Pharaun85** [comment]
Thanks for your response. 
I also thought about it at the time and tried to pass the tensors to NumPy arrays but it didn't work. I don't know how something so easy failed me, but now I've tried it again and it works.
